packages/p2p/src/components/my-profile/my-profile-stats/my-profile-name/my-profile-name.scss

```diff
@@ -77,20 +81,12 @@
         }
     }
 
-    &--row {
+    &__row {
```

Comment:
Renamed `my-profile-name--row` to `my-profile-name__row` for consistency, since `my-profile-name--rating` uses `my-profile-name--rating__row`